build/AzurePipelinesTemplates/MUX-NugetReleaseTest-Job.yml

```diff
@@ -0,0 +1,97 @@
+parameters:
+  dependsOn: ''
+  useNupkgFromArtifacts: false
+  nugetConfigPath: test\MUXControlsReleaseTest\nuget.config
+  matrix: 
+      Debug_x86:
+        buildPlatform: 'x86'
+        buildConfiguration: 'Debug'
+      
+jobs:
+- job: BuildNugetTests
+  dependsOn:
+    - ${{if parameters.dependsOn }}:
+      - ${{ parameters.dependsOn }}
+
+  pool:
+    vmImage: 'VS2017-Win2016'
+  strategy:
+    maxParallel: 10
+    matrix: ${{ parameters.matrix }}
+
+  variables:
+    solutionPath: test\MUXControlsReleaseTest\MUXControlsReleaseTest.sln
+    nugetConfigPath: test\MUXControlsReleaseTest\nuget.config
+    packageSaveDirectory: $(Build.SourcesDirectory)\packages\muxreleasetest
+    artifactDownloadPath: $(Build.SourcesDirectory)\Artifacts
+
+  steps:
+  # When using the nupkg from artifacts we need to make a new nuget.config that points the MUXReleaseTest
+  # projects at the just-built nupkg. The MUXRelease projects use a specific nuget version but the nuget
+  # restore logic is such that if the feeds provided don't include the requested version but instead have
```

Comment:
Can we update the test app to use packages.config and then just rewrite that?

Reply:
Or if not, can we update the proj to get the version number from a .props file and rewrite that?

Reply:
I was trying to cheat the way that the internal pipeline used to work but it was too fragile, so I've rewritten it to update the versions of the files themselves and not have any shenanigans.